source/schema/schema-management-upgrade.txt

```diff
@@ -0,0 +1,206 @@
+=========================
+Schema Management Upgrade
+=========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+The |bi| version 2.11 release introduces significant improvements to
+stored schema management. The new stored-schema format is
+equivalent to :ref:`DRDL <schema-with-drdl-file>` files. These
+improvements introduce some backwards-breaking changes for certain
```

Comment:
"backwards-breaking" is a bit jargon-y. Could rephrase as "some changes which are incompatible with previous versions"